src/components/views/location/LocationPicker.tsx

```diff
@@ -190,7 +190,8 @@ class LocationPicker extends React.Component<IProps, IState> {
                 position.coords.longitude +
                 ( position.coords.altitude != null ?
                     `,${ position.coords.altitude }` : '' ) +
-                `;u=${ position.coords.accuracy }`);
+                ( position.coords.accuracy != null ?
+                    `;u=${ position.coords.accuracy }` : '' );
```

Comment:
someone is going to walk along and change these to `!==` and break this, it really quite brittle to be checking for undefined using `null`

Reply:
I did wonder if this was the expected pattern...I can just change both

Reply:
(You should make this a linter rule btw, I'm a big fan of never using `!=`.)

Reply:
Fix'd, I believe these can only be number or undefined in the code?